spec/factories/events.rb

```diff
@@ -217,6 +217,46 @@
         create_list(:application_letter_canceled, evaluator.canceled_application_letters_count, event: event)
         create_list(:application_letter_pending, evaluator.pending_application_letters_count, event: event)
       end
+
+      trait :applications_with_profile do
+        after(:create) do |event|
+          event.application_letters.each {|application| application.user.profile = FactoryGirl.create(:profile) }
+        end        
+      end 
+    end
+
+    factory :event_in_execution_with_applications_in_various_states do
+      name "Event-Name"
+      description "Event-Description"
+      max_participants 20
+      date_ranges { build_list :date_range, 1 }
+      transient do
+        accepted_application_letters_count 1
+        rejected_application_letters_count 1
+        alternative_application_letters_count 1
+        canceled_application_letters_count 1
+      end
+      organizer "Workshop-Organizer"
+      knowledge_level "Workshop-Knowledge Level"
+      application_deadline Date.current
+
+      after(:create) do |event, evaluator|
```

Comment:
Is there a reason to use `create` instead of `build`? With `create` I think the line 252 with the `event.save!` could be even removed.